### ISSUE TITLE
Google rejected my app

### ISSUE THREAD

**Issue by amedeopro** [robodigital/cobrowseio_flutter#12]
Google rejected my app because your cobrowse.io package uses 
```
<service
            android:name="io.cobrowse.CobrowseAccessibilityService"
            android:permission="android.permission.BIND_ACCESSIBILITY_SERVICE" />
```
I also had to send them a video where I showed that the authorization requests are shown to the user who can also refuse to have a device connected remotely but apparently it didn't help, how can I do it? A thousand thanks

**Comment by HGray384** [comment]
@robinbonnes I have helped the customer via email as they reached out to us directly too (and it is a native SDK feature!). My understanding is that the reference to the accessibility service (which causes this) is optional in your plugin from the README docs.

It might be worth linking to our documentation in your README for full device remote control on android specifically as this does result in extra considerations for apps uploaded to the Google Play Store that we have already documented. Here is the link for reference: https://docs.cobrowse.io/sdk-features/full-device-capabilities/full-device-remote-control#configuring-full-device-remote-control

Hope that makes sense!
